### ISSUE TITLE
Accessibility Issue: Authenticator Component - Inaccessible Create Account and Sign In Tabs

### ISSUE THREAD

**Issue by gdelacampa** [aws-amplify/amplify-ui#6532]
### Before creating a new issue, please confirm:

- [x] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [x] I have tried disabling all browser extensions or using a different browser
- [x] I have tried deleting the node_modules folder and reinstalling my dependencies
- [x] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

React

### Which UI component?

Authenticator

### How is your app built?

Vite

### What browsers are you seeing the problem on?

Chrome, Firefox, Microsoft Edge

### Which region are you seeing the problem in?

us-west-2

### Please describe your bug.

We are using the out-of-the-box Authenticator component from AWS Amplify (v6.14.3). During an accessibility audit, we discovered that users are unable to navigate to the "Create Account" tab using the keyboard when they are on the "Sign-In" tab, and vice versa.

Upon inspecting the elements in the browser's developer tools, we observed that the tabindex attribute is set to -1 for the button that is not currently selected. This behavior prevents users from accessing the tabs via keyboard navigation, which is a significant accessibility issue. There does not appear to be a native way to override this behavior.

### What's the expected behaviour?

The Authenticator component should allow users to navigate between the "Sign-In" and "Create Account" tabs using the keyboard.

### Help us reproduce the bug!

To reproduce the bug, create a login page that imports the Authenticator component. Start the react app and navigate to the Authenticator component. Attempt to tab into the Create Account tab while on the Sign-In tab. 

Relevant dependencies are as follows:
    "@aws-amplify/ui-react": "^6.11.1",
    "aws-amplify": "^6.14.3",
    "react": "^18.3.1",
    "vite": "^6.2.6",

### Code Snippet

```javascript
import { Authenticator } from "@aws-amplify/ui-react";
export function AmplifyLogin() {
const services = ...;
const components = ...;
const formFields = ...;
 return (
    <div className="auth-wrapper pt-3">
      <Authenticator
        services={services}
        components={components}
        formFields={formFields}
      ></Authenticator>
    </div>
  );
}
```

### Additional information and screenshots

![Image](https://github.com/user-attachments/assets/4791621c-5f8f-46c5-ae70-0a543a513387)

**Comment by ashika112** [comment]
@gdelacampa , Checking this, Tab is not used for navigating here by design. But Customer should be able to navigate using arrow keys  once they are on the heading level and this should meet accessibility requirements. Can you please confirm if you are not seeing this behavior or if there further reasoning.
